src/components/Person/Person.jsx

```diff
@@ -1 +1,22 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export const Person = ({ person }) => {
+  const { name, age, sex, isMarried, partnerName } = person;
+  const married = sex === 'm'
+    ? 'wife'
+    : 'husband';
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">{`My name is ${name}`}</h2>
+      {age ? (
+        <p className="Person__age">{`I am ${age}`}</p>
+      ) : (null)}
+      {isMarried ? (
```

Comment:
Nested ternary operator is a bad practice. Try to write this logic in another way.

Reply:
Also when you use ternary operator line should start from "?" and ":" like this:
condition
  ? if_true
  : if_false